gitbook-docs/market/front-end.md

```diff
@@ -1,137 +1,137 @@
 ---
-description: Proof Market front end guide
+description: The web interface of the Proof Market
 ---
 
-# Front End
+# Web interface
 
 {% hint style="info" %}
 Please note this product is in beta and expected to change.
 {% endhint %}
 
-#### Proof Market front end can be accessed at [**proof.market**](https://proof.market)
+#### The web interface of the Proof Market can be accessed at [**proof.market**](https://proof.market).
 
 ## New user signup
 
-All users who wish to interact with the proof market must create an account.
+All users who wish to interact with the Proof Market must create an account.
 
 {% hint style="info" %}
-If you do not have the credentials, users can still view the exchange, but they will be unable to post orders to buy/sell proofs.
+If you do not have an account, you can still view the exchange, but you won't be able to post orders
+to buy or sell proofs.
 {% endhint %}
 
-<figure><img src="../.gitbook/assets/image (6).png" alt=""><figcaption></figcaption></figure>
+To register a new user via the web interface, click on **Sign up**:
 
-Click on Sign up
+<figure><img src="../.gitbook/assets/image (6).png" alt=""><figcaption><p>Sign up view</p></figcaption></figure>
 
-<figure><img src="../.gitbook/assets/image.png" alt=""><figcaption></figcaption></figure>
+Enter your username and password and complete the flow by clicking on **Register**.
 
-Select your username and password and complete the flow by clicking register.
+<figure><img src="../.gitbook/assets/image.png" alt=""><figcaption><p>Registration view</p></figcaption></figure>
 
+## Login
 
+If you're a registered user, enter your credentials in the provided fields and click **Login**.
 
-## Login&#x20;
-
-<figure><img src="../.gitbook/assets/image (7).png" alt=""><figcaption><p>Login</p></figcaption></figure>
-
-Authenticated users should enter their credentials in the provided fields. &#x20;
+<figure><img src="../.gitbook/assets/image (7).png" alt=""><figcaption><p>Login view</p></figcaption></figure>
 
 ## Dashboard
 
-<figure><img src="../.gitbook/assets/image (11).png" alt=""><figcaption></figcaption></figure>
-
-The proof market dashboard can be broken down into the following components
-
-* [A: Circuit List](front-end.md#circuit-list)
-* [B:  Trades](front-end.md#trades)
-* [C: Last Proof Producer](front-end.md#last-proof-producer)
-* [D: Circuit Details](front-end.md#circuit-details)
-* [E: Circuit Price Dashboard](front-end.md#circuit-price-dashboard)
-* [F: Order Book](front-end.md#order-book)
-* [G: Create Orders](front-end.md#create-orders)
-* [H: Manage Orders](front-end.md#manage-orders)
+<figure><img src="../.gitbook/assets/image (11).png" alt=""><figcaption><p>The Proof Market dashboard</p></figcaption></figure>
 
-### Circuit List
+The Proof Market dashboard can be broken down into the following components:
```

Comment:
Somewhy I don't like this "broken down" used with our service that we actually don't want to break down. Let's say that it has the following components.